index.js

```diff
@@ -140,29 +144,9 @@ function onWorkerMessage (msg) {
       // Replace the FakeWeakRef with a
       // proper one.
       this.stream = new WeakRef(stream)
-      if (stream._sync) {
-        stream.ready = true
-        stream.flushSync()
-        stream.emit('ready')
-      } else {
-        stream.once('drain', function () {
-          stream.flush(() => {
-            stream.ready = true
-            stream.emit('ready')
-          })
-        })
-        nextFlush(stream)
-      }
```

Comment:
Please restore emitting "ready".

Reply:
What's the use case?

Reply:
Detecting errors early. Pino transport currently relies on this to verify everything is in order.